### ISSUE TITLE
Exported then reloaded Classifier model gives wrong prediction

### ISSUE THREAD

**Issue by winston-p** [keras-team/autokeras#1447]
EDIT: I just realised that what I thought were predicted labels are actually classification probabilities. There is no bug then and I have closed this issue.

### Bug Description
I am trying out AutoKeras by following the step-by-step instructions in [https://autokeras.com/tutorial/export/](url). Everything went fine until `print(predicted_y)` produced:

```
[[7.10387882e-09 5.58982416e-10 3.74930835e-07 ... 9.99997973e-01
  2.01310062e-08 3.60369455e-07]
 [1.85361150e-05 9.04550598e-06 9.99895692e-01 ... 2.75132152e-12
  3.61683783e-06 7.38385242e-10]
 [4.39638507e-06 9.98704195e-01 1.83042779e-04 ... 3.79047066e-04
  8.97390855e-05 2.85495821e-06]
 ...
 [2.76643597e-09 3.89823036e-08 2.50714938e-09 ... 1.26030145e-05
  4.11345856e-04 1.28301617e-04]
 [1.66736356e-07 1.93144473e-10 1.16833530e-08 ... 4.25922603e-10
  3.32917640e-04 2.25114619e-07]
 [1.36902031e-06 2.86963953e-10 1.59475476e-05 ... 1.64523464e-11
  2.79402485e-07 4.60360372e-09]]
```

I then tried `print(clf.predict(x_test))` and that gave the correct-looking results:
```
[['7']
 ['2']
 ['1']
 ...
 ['4']
 ['5']
 ['6']]
```

### Bug Reproduction
Code for reproducing the bug: As per [https://autokeras.com/tutorial/export/](url)

Data used by the code: MNIST data loaded using `(x_train, y_train), (x_test, y_test) = mnist.load_data()`

### Expected Behavior
Predicted results should be a matrix with 1 column and integer datatype.

### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu 18.04 LTS
 - Python: 3.6.9
 - autokeras: 1.0.11
 - keras-tuner: 1.0.2rc4
 - scikit-learn: 0.23.2
 - numpy: 1.18.5
 - pandas: 1.1.4
 - tensorflow: 2.3.1
 - tensorboard: 2.4.0
 - tensorflow-estimator: 2.3.0

### Additional context
- I tested the above by installing keras-tuner v1.0.2 but the issue still persists.
- I am running AutoKeras with an Nvidia T4 GPU, with Nvidia driver v450.80.02, CUDA v10.1 and cuDNN v7_7.6.5.32 installed

**Comment by andreaspapav** [comment]
I still have the same question on how to include the labels rather than the probabilities of an exported model

**Comment by haifeng-jin** [comment]
you have to do your own post-processing, it is not supported currently by AutoKeras.
